### PR TITLE
connection plugins: use f-strings

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY

Use f-strings instead of string interpolations or string format().

##### ISSUE TYPE

- Refactoring Pull Request

##### COMPONENT NAME

plugins/connection/chroot.py
plugins/connection/funcd.py
plugins/connection/incus.py
plugins/connection/iocage.py
plugins/connection/jail.py
plugins/connection/lxc.py
plugins/connection/lxd.py
plugins/connection/qubes.py
plugins/connection/saltstack.py
plugins/connection/zone.py

### CODE REVIEW

File: plugins/connection/qubes.py [hunk 1]
```diff
@@ -114,42 +114,42 @@ def exec_command(self, cmd, in_data=None, sudoable=False):
         """Run specified command in a running QubesVM """
         super(Connection, self).exec_command(cmd, in_data=in_data, sudoable=sudoable)
 
-        display.vvvv("CMD IS: %s" % cmd)
+        display.vvvv(f"CMD IS: {cmd}")
 
         rc, stdout, stderr = self._qubes(cmd)
 
-        display.vvvvv("STDOUT %r STDERR %r" % (stderr, stderr))
+        display.vvvvv(f"STDOUT {stderr!r} STDERR {stderr!r}")
```
Comment (felixfontein) [inline plugins/connection/qubes.py:121]: This looks like a bug, it's printing stderr twice. (Not related to this PR though.)

Comment (russoz) [reply plugins/connection/qubes.py:121]: I'd rather fix the bug first

Comment (russoz) [reply plugins/connection/qubes.py:121]: addressed in #9334

Comment (russoz) [reply plugins/connection/qubes.py]: rebased!

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks!

**Pull request merged**